R/format.R

```diff
@@ -90,11 +143,139 @@ xportr_format <- function(.df,
 
   names(format) <- filtered_metadata[[variable_name]]
 
+  # vector of expected formats for clinical trials (usually character or date/time)
+  # https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/leforinforref
+  # /n0p2fmevfgj470n17h4k9f27qjag.htm#n0wi06aq4kydlxn1uqc0p6eygu75
+
+  expected_formats <- c(
```

Comment:
Yes the `utils-xportr.R` expected formats currently has mistakes in it. No this code won't necessarily need removing - it would just mean that the checks are carried out once in `xportr_format()` and once in `xportr_write()`.

Unless of course you would then want this removed?

Reply:
We should just have one source of truth.  Since you have the right formats can you just replace in the utils file?  I know we talked about splitting out this into different PRs but I did t fully understand on how it would impact.  Apologies

Reply:
Sure! I'll link those other two issues to this PR then also.

Reply:
Thanks Sophie - since the correct formats are now in `utils-xportr.R` can this function just call that file for the formats?  That way we just have one place to maintain them.

Reply:
Good idea @averissimo - I will have a go at that.